llarp/net/uint128.hpp

```diff
@@ -141,25 +141,25 @@ namespace llarp
     constexpr bool
     operator<(const uint128_t& b) const
     {
-      return upper < b.upper || (upper == b.upper && lower < b.lower);
+      return std::tie(upper, lower) < std::tie(b.upper, b.lower);
     }
 
     constexpr bool
     operator<=(const uint128_t& b) const
     {
-      return upper < b.upper || (upper == b.upper && lower <= b.lower);
+      return std::tie(upper, lower) <= std::tie(b.upper, b.lower);
     }
 
     constexpr bool
     operator>(const uint128_t& b) const
     {
-      return upper > b.upper || (upper == b.upper && lower > b.lower);
+      return std::tie(upper, lower) > std::tie(b.upper, b.lower);
     }
 
     constexpr bool
     operator>=(const uint128_t& b) const
     {
-      return upper > b.upper || (upper == b.upper && lower >= b.lower);
+      return std::tie(upper, lower) >= std::tie(b.upper, b.lower);
```

Comment:
(These changes aren't changing anything, but rather just making it consistent with all the other inequality comparisons in this PR).